addons/amb_civ_command/fnc_civCommandRouter.sqf

```diff
@@ -386,10 +386,10 @@ switch(_operation) do {
 
     case "stopManagement": {
 
-        private _debug = _logic select 2 select 0;
-        private _handle = _logic select 2 select 3;
+        private _debug = [_logic,"debug",false] call ALIVE_fnc_hashGet;
+        private _handle = [_logic,"managerHandle",objNull] call ALIVE_fnc_hashGet;
```

Comment:
`scriptNull` instead ?

Reply:
spawnhandle != scriptNull
but it is better for the understanding of the type... in the end it is imporant that it is a null type ^^